README.md

```diff
@@ -114,7 +116,7 @@ Any other original works found to be not explicitly covered by the above definit
 
 **Q: Can I download the aircraft in the current state?**
 
-A: Yes, see [Downloads](#Downloads).
+A: Yes, please seek [Downloads](#Downloads).

```

Comment:
This change doesn't make any sense. What does it mean to "seek Downloads"?

Reply:
Thanks for taking time out of your day to notice my mistakes that I made while correcting others. I will take a bit more time and fix them up to make sure they fit perfectly.